pkg/app/piped/cloudprovider/kubernetes/kubectl.go

```diff
@@ -70,6 +76,74 @@ func (c *Kubectl) Apply(ctx context.Context, namespace string, manifest Manifest
 	return nil
 }
 
+func (c *Kubectl) Create(ctx context.Context, namespace string, manifest Manifest) (err error) {
+	defer func() {
+		kubernetesmetrics.IncKubectlCallsCounter(
+			c.version,
+			kubernetesmetrics.LabelCreateCommand,
+			err == nil,
+		)
+	}()
+
+	data, err := manifest.YamlBytes()
+	if err != nil {
+		return err
+	}
+
+	args := make([]string, 0, 5)
+	if namespace != "" {
+		args = append(args, "-n", namespace)
+	}
+	args = append(args, "create", "-f", "-")
+
+	cmd := exec.CommandContext(ctx, c.execPath, args...)
+	r := bytes.NewReader(data)
+	cmd.Stdin = r
+
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("failed to create: %s (%v)", string(out), err)
```

Comment:
From the new go version, `%w` is more suitable for `fmt.Errorf` since the caller will be able to unwrap the included error.

```suggestion
		return fmt.Errorf("failed to create: %s (%w)", string(out), err)
```

Reply:
Thank you! 
Should I also change Apply (L74)?

Reply:
Yes. Please. 💯